src/main/java/com/laserfiche/api/client/httphandlers/UsernamePasswordHandler.java

```diff
@@ -0,0 +1,74 @@
+package com.laserfiche.api.client.httphandlers;
+
+import com.laserfiche.api.client.APIServer.SessionKeyInfo;
+import com.laserfiche.api.client.APIServer.TokenClient;
+import com.laserfiche.api.client.APIServer.TokenClientImpl;
+import com.laserfiche.api.client.model.CreateConnectionRequest;
+
+import java.util.concurrent.CompletableFuture;
+
+public class UsernamePasswordHandler implements HttpRequestHandler {
+    private String _accessToken;
```

Comment:
would you please consider removing the leading _, as it is not a common pattern in java.

Reply:
done